frontend/src/components/phones/dashboard/PhoneWelcomeView.module.scss

```diff
@@ -0,0 +1,188 @@
+      + .instruction-item {
+        border-top: solid 2px $color-light-gray-20;
+
+        @media screen and #{$mq-md} {
+          border-left: solid 2px $color-light-gray-20;
+          border-top: none;
+        }
+      }
+
+      h2 {
+        @include text-title-3xs;
+        padding: $spacing-md 0;
+        font-weight: 600;
+      }
+
+      .welcome-text-cta {
+        display: inline-block;
+        margin: 0;
+        padding: 0;
+        text-decoration: underline;
+        color: $color-blue-50;
+        cursor: pointer;
+        font-weight: 500;
+        background: none;
+        border: none;
+
+        &:hover {
+          color: $color-blue-80;
+        }
+      }
+
+      img {
+        align-self: center;
+      }
+
+      .demo-wrapper {
+        padding: 0 $spacing-md;
+
+        @media screen and #{$mq-md} {
+          padding: 0;
+        }
+
+        .demo-heading {
+          color: $color-light-gray-80;
+          padding-top: $spacing-lg;
+        }
+
+        .demo-input-wrapper {
+          display: flex;
+          align-items: center;
+          position: relative;
+
+          .demo-input {
+            @include text-body-lg;
+            position: absolute;
+            // Positioning the input into the text image
+            margin: 0 0 5% 5%;
+
+            @media screen and #{$mq-md} {
+              @include text-body-md;
+            }
+            @media screen and #{$mq-xl} {
+              @include text-body-lg;
+            }
+          }
+
+          .demo-img {
+            width: 100%;
+            // To counteract the drop shadow in the text image
+            margin-left: -8px;
+
+            img {
+              width: 100%;
+            }
+          }
+        }
+
+        .block-sender-wrapper {
+          width: 100%;
+
+          ul {
+            display: flex;
+            flex-direction: column;
+            padding: $spacing-sm;
+            background: $color-white;
+            box-shadow: 0px 2px 12px rgba(0, 0, 0, 0.1); // Taken from Figma style to match it with the other svgs
+            border-radius: $border-radius-md;
+            list-style: none;
+            gap: $spacing-xs;
+
+            li {
+              display: flex;
+              flex-direction: row;
+              align-items: center;
+              justify-content: space-between;
+              padding: $spacing-xs;
+
+              &:first-child {
+                color: $color-light-gray-50;
+                padding-bottom: $spacing-sm;
+                border-bottom: 1px solid $color-light-gray-10;
+              }
+
+              span {
+                margin: 0;
+
+                p:nth-child(2) {
+                  @include text-body-sm;
+                }
+
+                &:nth-child(2) {
+                  color: $color-blue-50;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .dashboard-btn {
+    @include font-firefox;
+    margin-top: $layout-md;
```

Comment:
Referencing back to the composable CSS article from a while ago, instead of adding a `margin-top` to both `.dashboard-btn` and `.phone-instructions-wrapper`, it's safer to add a `gap: $layout-md` to the parent `main-wrapper`.